lib/testmap.py

```diff
@@ -27,16 +27,16 @@
     },
     'cockpit-project/cockpit': {
         'main': [
-            'fedora-34/container-bastion',
+            'fedora-35/container-bastion',
             'debian-stable',
             'debian-testing',
             'ubuntu-2004',
             'ubuntu-stable',
             'fedora-34',
-            'fedora-34/devel',
-            'fedora-34/firefox',
-            'fedora-34/mobile',
-            'fedora-35',
+            TEST_OS_DEFAULT,
```

Comment:
I'd maybe avoid this particular one and just leave it open.

Reply:
Ok, done.